src/components/personalSettings/profileSettings/profileSettings.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+
+import style from './profileSettings.module.css';
+import { useForm } from '../../formValidation/index';
+import { connect } from 'react-redux';
+import { setUserDataThunkCreator } from '../../../redux/auth-reducer';
+
+function ProfileSettings(props) {
+  const initialState = {
+    firstname: '',
+    lastname: '',
+    dateOfBirth: '',
+    country: '',
+    gender: ''
+  };
+  function Submit() {
```

Comment:
```suggestion
  function submit() {
```
all function names need to start with lower case